scripts/weblate/double_languages.py

```diff
@@ -1,45 +1,47 @@
-
-
 import wlc
 import apikey
 import requests
 import math
-import json
 from requests.exceptions import HTTPError
+import json
 
 
 w=wlc.Weblate(apikey.myAPIKey)
 projects={'mobile':{},'server':{},'webapp':{},'glossary':{}}
-projects['mobile']['shipped']='mattermost-mobile_master'
+projects['mobile']['shipped']='mattermost-mobile-v2'
 projects['mobile']['wip']='mattermost-mobile-wip'
 projects['server']['shipped']='mattermost-server_master'
 projects['server']['wip']='mattermost-server-wip'
 projects['webapp']['shipped']='mattermost-webapp_master'
 projects['webapp']['wip']='mattermost-webapp-wip'
 projects['glossary']['shipped']='glossary'
 projects['glossary']['wip']='glossary'
+
 current_notification_state={"state":"false"}
 previous_notification_state= json.load(open("./notification_state.txt"))
 
 for project in projects:
   shippedLanguages={}
   WIPLanguages={}
-### GETTING THE SHIPPED LANGUAGES ###
-  #print('GETTING SHIPPED LANGUAGES FOR '+project)
-  shippedProjects=w.get('https://translate.mattermost.com/api/components/mattermost/'+projects[project]['shipped']+'/translations/')
-  for shippedLanguage in shippedProjects['results']:
-    if shippedLanguage['language']['code']=='en':
-      continue
-    shippedLanguageCode=shippedLanguage['language']['code']
-    shippedLanguageName=shippedLanguage['language']['name']
-    shippedLanguages[shippedLanguageCode]=shippedLanguageName
+  page=1
+  next='https://translate.mattermost.com/api/components/mattermost/'+projects[project]['shipped']+'/translations/'
+  shippedProjects=w.get(next)
+  lastpage=math.ceil(shippedProjects['count']/20)  
+
+  while page<=lastpage:
+    page=page+1
+    for shippedLanguage in shippedProjects['results']:
+      if shippedLanguage['language']['code']=='en':
+        continue
+      shippedLanguageCode=shippedLanguage['language']['code']
+      shippedLanguageName=shippedLanguage['language']['name']
+      shippedLanguages[shippedLanguageCode]=shippedLanguageName
```

Comment:
Apologies if my Python is rusty, but I don't actually see a request to /get/ the next page: it seems like the code just iterates over `shippedProjects` fetched from the first page above.

Reply:
this was also solved with the new iterator